AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -2475,7 +2474,7 @@ decathlon.hu###main-content-container > #container-screen:has(> #container-cooki
 ~www.wp.pl,~poczta.wp.pl,wp.pl###app > div > table > tbody > tr:nth-child(2) > td > span > div:has(> div[class] > div[class] > a[href="http://onas.wp.pl/poufnosc.html"])
 pugski.com##.Panels > .Notice.DismissParent:has(> .noticeContent > .noticeCookiesContent)
 forum.gamer.com.tr##.p-body-inner > .notices > .notice:has(> .notice-content > a[href="/help/cookies"])
-bethesda.net##app > visor-alert[name="visor-alert"] > div[class=""] > div[class^="jss"] > div[class^="jss"]:has(> a[href="/document/cookie-policy"])
+bethesda.net##app > visor-alert[name="visor-alert"] > div[class=""] > div[class^="jss"] > div[class^="jss"]:has(> a[href$="/document/cookie-policy"])
```

Comment:
removed rule covers it

Reply:
yes, but removed rule blocks useful popups like wrong password notification (that was the issue)